contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+### Introduction
+
+This document focuses on cStor day 2 operations work flow via the proposed specs
+of CSPC and CSP mentioned in contribute/design/cstor-operator/specs.md
+
+### Pool Expansion
+
+* “As a OpenEBS user, I should be able to add block devices to CSPC to increase
+   the pool capacity.” *
+
+Steps To Be Performed By User:
+
+1. `kubectl edit cspc your_cspc_name`
+
+2. Put the block devices in the CSPC spec against the correct kubernetes nodes.
+
+
+**Consider following example for stripe pool expansion**
+
+Current CSPC is following:
+
+
+This CSPC corresponds to a stripe pool on node `gke-cstor-it-default-pool-1`
+
+```
+
+apiVersion: openebs.io/v1alpha1
+kind: CStorPoolCluster
+metadata:
+  name: cstor-pool-stripe
+spec:
+  type: stripe
+  nodes:
+
+  - hostName: gke-cstor-it-default-pool-1
```

Comment:
convert this into `hostName` to `nodeSelector` and then have an array under it that selects the exact node. For example:

```
  nodes:
    - nodeSelector:
        kubernetes.io/hostname: gke-cstor-it-default-pool-1
```